e2e/test/scenarios/dashboard/actions-on-dashboards.cy.spec.js

```diff
@@ -75,29 +77,47 @@ const MODEL_NAME = "Test Action Model";
 
           cy.findByRole("dialog").within(() => {
             fillActionQuery(
-              `INSERT INTO ${TEST_TABLE} (team_name) VALUES ('Zany Zebras')`,
+              `UPDATE ${TEST_TABLE} SET score = {{ new_score }} WHERE id = {{ id }}`,
```

Comment:
instead of adding a new e2e test, I beefed up this existing one to test more aspects of actions.